src/main/java/org/fundacionjala/coding/maria/Kata1.java

```diff
@@ -0,0 +1,33 @@
+package org.fundacionjala.coding.maria;
+
+/**
+ * kata 1 Spin Words.
+ * Inverted String that have more of 5 letters
+ * Author: Maria Canqui
+ */
+public class Kata1 {
+    private static final int TAM = 5;
+
+    /**
+     * @param words This is the sentence
+     * @return words A new inverted sentence
+     */
+    public String spinWords(String words) {
+        String auxString = words;
+        String[] wordsList;
+        wordsList = auxString.split(" ");
+        auxString = "";
+        StringBuffer buf = new StringBuffer();
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Replace the synchronized class "StringBuffer" by an unsynchronized one such as "StringBuilder". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1149)

Reply:
For this case it would be better using a StringJoiner